CRM/Event/Form/Search.php

```diff
@@ -90,31 +90,8 @@ public function preProcess() {
     $this->_done = FALSE;
     $this->defaults = array();
 
-    /*
-     * we allow the controller to set force/reset externally, useful when we are being
-     * driven by the wizard framework
-     */
-    $this->_reset = CRM_Utils_Request::retrieve('reset', 'Boolean');
-    $this->_force = CRM_Utils_Request::retrieve('force', 'Boolean', $this, FALSE);
-    $this->_limit = CRM_Utils_Request::retrieve('limit', 'Positive', $this);
-    $this->_context = CRM_Utils_Request::retrieve('context', 'Alphanumeric', $this, FALSE, 'search');
-    $this->_ssID = CRM_Utils_Request::retrieve('ssID', 'Positive', $this);
-    $this->assign("context", $this->_context);
-
-    // get user submitted values
-    // get it from controller only if form has been submitted, else preProcess has set this
-    if (!empty($_POST) && !$this->controller->isModal()) {
-      $this->_formValues = $this->controller->exportValues($this->_name);
-    }
-    else {
-      $this->_formValues = $this->get('formValues');
-    }
-
-    if (empty($this->_formValues)) {
-      if (isset($this->_ssID)) {
-        $this->_formValues = CRM_Contact_BAO_SavedSearch::getFormValues($this->_ssID);
-      }
-    }
+    $this->getUrlVariables();
+    $this->getFormValues();
```

Comment:
@eileenmcnaughton, I was trying to do some local search/replace on the new function names, and this stood out as an oddity:

* The patch to `CRM/Core/Form/Search.php` adds `getFormVariables()`.
* The various search classes call a mix of `getFormValues()` or `getFormVariables()`.
* In at least two cases (`CRM/{Case,Event}/Form/Search.php`), there seems to be a pre-existing function named `getFormValues()`  which is just a null-op (`return NULL`).  They don't match up with the extracted code, so this smells like a change in the behavior.

(When I hit this realization, I gave up on locally renaming the functions myself and just documented instead.)